File: Java/src/pl/eco/zwierzeta/Wilk.java
package pl.eco.zwierzeta;

import pl.eco.Swiat;
import pl.eco.Zwierze;

public class Wilk extends Zwierze {
    public Wilk(int x, int y, Swiat moj) {
        sila = 9;
        inicjatywa = 5;
        symbol = 'W';
        nazwa = "wilk";
        wiek = 0;
        poRozmnozeniu=0;
        this.x = x;
        this.y = y;
        this.moj = moj;
    }
    public Wilk(int x, int y, Swiat moj, int Sila, int Wiek, int poRozmnazaniu) {
        this.sila = sila;
        this.wiek = Wiek;
        this.poRozmnozeniu=poRozmnazaniu;
        inicjatywa = 5;
        nazwa = "wilk";
        symbol = 'W';
        this.x = x;
        this.y = y;
        this.moj = moj;
    }
}

File: Python/Zwierzeta/Wilk.py
from Zwierze import Zwierze

class Wilk (Zwierze):
    def __init__(self, x, y, moj):
        self.sila = 9
        self.inicjatywa = 5
        self.symbol = 'W'
        self.nazwa = "wilk"
        self.wiek = 0
        self.poRozmnozeniu = 0
        self.x = x
        self.y = y
        self.moj = moj
        self.czyZyje = True

    def generator(self, x, y):
        return Wilk(x, y, self.moj)